redaxo/src/addons/be_style/scss/_mime.scss

```diff
@@ -18,10 +19,13 @@
     text-transform: uppercase;
 }
 .rex-mime-default {
+    @extend .fa-regular;
+    @extend .fa-file;    
```

Comment:
Diese zwei Zeilen sind überflüssig, da bereits in Zeile 1 notiert

Reply:
erledigt (kommt mit nächster Version des PR)

Reply:
https://github.com/redaxo/redaxo/pull/5712/commits/6e86b5ae5be6512acb73739967487b9dffff8dee